docs/Account sign up (Account Protection) - Approved and rejected sign up flows.md

```diff
@@ -0,0 +1,126 @@
+# Microsoft Dynamics 365 Fraud Protection - API examples
```

Comment:
I would say most of my feedback on the sign in doc applies on this sign up doc, too.